PRINCIPLES.md

```diff
@@ -55,7 +55,7 @@ _Principle: Great projects already have many ingredients to succeed. First: do n
 Identify projects that have a real shot at being a useful tool in the evolving box of cloud native technology. This is a mix of mature and early-stage projects.   Early stage may not have all the criteria we want: diverse contributor bases, formalized governance, interoperability, cloud-native designs, quality bar, etc. 
 
 Some considerations:
-- Transparent, consistent technical and governance quality bar for [graduation](https://www.cncf.io/projects/graduation-criteria/) from incubation
+- Transparent, consistent technical and governance quality bar for [graduation](https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc) from incubation
```

Comment:
the correct link is`(/process/graduation_creteria.adoc/)` I believe. There is no need to specify the full URI and this reduces load on any future listing for local docs. Additionally all links should conclude with a `/` unless they include a `#` hash to reduce double hits on the web server.